lib/obs_github_deployments/cli/commands.rb

```diff
@@ -5,6 +5,7 @@ module CLI
     module Commands
       extend Dry::CLI::Registry
       # register the commands and its command line
+      register "is_locked", IsLocked, aliases: ["i", "-i", "--is-locked"]
```

Comment:
For consistency, I suggest to use hyphen (-) as we do in `set-lock`.

Reply:
I wouldn't use `i` and `-i` as aliases, as they usually refer to something else in well-known CLIs.

This guideline explains why: https://clig.dev/#arguments-and-flags

Reply:
:+1:

Reply:
As talked offline, we won't use an alias for this command